source/dotnet/Library/AdaptiveCards.Rendering.Wpf.Xceed/XceedDateInput.cs

```diff
@@ -40,6 +40,12 @@ public static FrameworkElement Render(AdaptiveDateInput input, AdaptiveRenderCon
                 datePicker.Style = context.GetStyle("Adaptive.Input.Date");
                 datePicker.DataContext = input;
 
+                if (input.IsRequired && input.ErrorMessage == null)
+                {
+                    context.Warnings.Add(new AdaptiveWarning((int)AdaptiveWarning.WarningStatusCode.NoWarningForValidatedInput,
+                        "Inputs with validation should include an ErrorMessage"));
```

Comment:
@RebeccaAnne do we have a plan to localize these error strings that we bubble out via the AdaptiveWarning/Error object(s)?

Reply:
There's not currently any plan to localize warning strings. I added it to the doc i'm working on that covers possible areas for localization.

---
In reply to: [501253546](https://github.com/microsoft/AdaptiveCards/pull/4885#discussion_r501253546) [](ancestors = 501253546)